docs/0.1/index.md

```diff
@@ -810,11 +810,11 @@ A deleted entity _MUST_{:.strong-term} be implemented as an _Activity_{:.term} f
 ### A. Provider Workflows
 {: #provider-workflows}
 
-The section describes how an Entity Metadata Provider can implement this specification to allow consumer to follow changes in a set of entities they manage.
+The section describes how an Entity Metadata Provider can implement this specification to allow consumer to follow changes in a set of entities that they provider manages.
```

Comment:
typo: that the[~y~] provider manages

Reply:
The current convention in the doc seems to be capitalize Entity Metadata Provider and Entity Metadata Consumer, but lower case for provider and consumer when the fuller phrases aren't used. Do we want to capitalize every use of Provider and Consumer, or stick with the current convention?

Reply:
Might we use abbreviations?  For example, at the first naming use 'Entity Metadata Provider (EMP)' and 'Entity Metadata Consumer (EMC)' and then refer to them throughout the rest of the doc as EMP and EMC?  I'm not trying to be difficult, but it seems confusing to refer to them generically and indiscriminately as 'provider' and 'consumer' when we've made of point of identifying formal names for them.

Reply:
I think that readability is better without abbraviations